### PR TITLE
add ex13 files

### PR DESCRIPTION
**Pull request by norona-marynicole**
I add my ex13 file.

### CODE REVIEW

File: norona_marynicole/background_photo.jpg
Comment (MentokAlmera) [inline norona_marynicole/background_photo.jpg:1]: I suggest all your image files be placed in one folder so they are not scattered.

File: norona_marynicole/index.html [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+            name="full_name"
+            id="full_name"
+            placeholder="Full Name"
+          />
+          <input
+            type="text"
+            name="subject"
+            id="subject"
+            placeholder="Subject"
+          />
```
Comment (MentokAlmera) [inline norona_marynicole/index.html]: Remove unnecessary breaking of lines

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_27

File: norona_marynicole/index.html [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+          <input type="number" name="age" id="age" placeholder="Age" />
+          <label for="copy">Send me a copy</label>
+          <input type="checkbox" name="copy" id="copy" />
+          <textarea id="message" name="message" rows="3" cols="15">
+          Write a Message</textarea
+          >
```
Comment (MentokAlmera) [inline norona_marynicole/index.html]: Indent the write a message

Comment (MentokAlmera) [inline norona_marynicole/index.html]: </textarea>

File: norona_marynicole/style.css [hunk 3]
```diff
@@ -0,0 +1,162 @@
+form {
+  display: grid;
+  border: 1px solid;
+  padding: 10px;
+}
+
+#email #name #subject #age {
```
Comment (MentokAlmera) [inline norona_marynicole/style.css]: #email,
#name,
#subject,
#age

**Review by MentokAlmera — CHANGES_REQUESTED** [review]
After you put your img files into one folder, modify your html for your path in the img src.

File: norona_marynicole/index.html [hunk 4]
```diff
@@ -145,25 +145,22 @@ <h2 class="title">FORM</h2>
         <h2>Contact Me</h2>
 
         <form class="form">
-          <input type="email" name="email" id="email" placeholder="Email" />
+          <input type="email" name="email" id="email" placeholder="Email">
```
Comment (MentokAlmera) [inline norona_marynicole/index.html:148]: fix closing tag
/>

File: norona_marynicole/index.html [hunk 5]
```diff
@@ -145,25 +145,22 @@ <h2 class="title">FORM</h2>
...
           <input
             type="text"
             name="full_name"
             id="full_name"
-            placeholder="Full Name"
-          />
+            placeholder="Full Name">
```
Comment (MentokAlmera) [inline norona_marynicole/index.html:153]: fix closing tag
/>

File: norona_marynicole/index.html [hunk 6]
```diff
@@ -145,25 +145,22 @@ <h2 class="title">FORM</h2>
...
           <input
             type="text"
             name="subject"
             id="subject"
-            placeholder="Subject"
-          />
-          <input type="number" name="age" id="age" placeholder="Age" />
+            placeholder="Subject">
+          <input type="number" name="age" id="age" placeholder="Age">
```
Comment (MentokAlmera) [inline norona_marynicole/index.html:159]: fix closing tag
/>

File: norona_marynicole/index.html [hunk 7]
```diff
@@ -145,25 +145,22 @@ <h2 class="title">FORM</h2>
...
           <label for="copy">Send me a copy</label>
-          <input type="checkbox" name="copy" id="copy" />
+          <input type="checkbox" name="copy" id="copy">
```
Comment (MentokAlmera) [inline norona_marynicole/index.html:161]: fix closing tag
/>

File: norona_marynicole/index.html
Comment (MentokAlmera) [inline norona_marynicole/index.html:1]: modify the path of your img source 

Picture 1

**Review by MentokAlmera — CHANGES_REQUESTED** [review]
Please fix the closing tag and modify the imgae path.

**Review by MentokAlmera — APPROVED** [review]
Great! You can merge it now.

**Comment by norona-marynicole** [comment]
Thank you so much for guiding me po. Sorry po now lang mag merge, galing po kasi akong school eh.

**Pull request merged**